docs/DeveloperGuide.md

```diff
@@ -921,36 +921,242 @@ testers are expected to do more *exploratory* testing.
+
+    5. Other incorrect review edit commands to try: `redit`, `redit 1`, `redit x d/20/09/2022`, `redit x c/The chicken was dry`(where x is larger than the list size)<br>
+       Expected: Similar to previous.
+
+### Find stalls
+
+1. Finding stalls by name and/or tags
+
+    1. Prerequisites: At least one stall entry exists that we are finding with name and/or tag exists.
+   
+    2. Test case: `sfind n/chicken`<br>
+       Condition: There is a stall with the word `chicken` in its name<br>
+       Expected: Status message responds with the number of stalls that matches the name in command. GUI updates to show stalls that matches the name in command. 
+   
+    3. Test case: `sfind t/opendaily`<br>
+       Condition: There is a stall with the tag `opendaily`<br>
+       Expected: Status message responds with the number of stalls that matches the tag in command. GUI updates to show stalls that matches the tag in command.
+   
+    4. Test case: `sfind n/chicken t/opendaily`<br>
+       Condition: There is a stall with the word `chicken` in its name and with the tag `opendaily`<br>
+       Expected: Status message responds with the number of stalls that matches the name and tag in command. GUI updates to show stalls that matches the name and tag in command.
+   
+    5. Test case: `sfind n/Alice's`<br>
+       Expected: No stalls are found. Error details shown in the status message. GUI does not update.
+
+    6. Other incorrect stall find commands to try: `sfind`, `sfind n/`, `sfind a/Ang Mo Kio`<br>
+       Expected: Similar to previous.
+
+### Find reviews
+
+1. Finding reviews by stall name and/or tags
+
+    1. Prerequisites: At least one review entry exists that we are finding with stall name and/or tag exists.
+
+    2. Test case: `rfind n/chicken`<br>
+       Condition: There is a review with the word `chicken` in its stall name<br>
+       Expected: Status message responds with the number of reviews that matches the stall name in command. GUI updates to show reviews that matches the stall name in command.
+
+    3. Test case: `rfind t/delicious`<br>
+       Condition: There is a review with the tag `delicious`<br>
+       Expected: Status message responds with the number of reviews that matches the tag in command. GUI updates to show reviews that matches the tag in command.
+
+    4. Test case: `rfind n/chicken t/delicious`<br>
+       Condition: There is a review with the word `chicken` in its stall name and with the tag `delicious`<br>
+       Expected: Status message responds with the number of reviews that matches the stall name and tag in command. GUI updates to show reviews that matches the stall name and tag in command.
+
+    5. Test case: `rfind n/Alice's`<br>
+       Expected: No reviews are found. Error details shown in the status message. GUI does not update.
+
+    6. Other incorrect stall find commands to try: `rfind`, `rfind n/`, `rfind a/Ang Mo Kio`<br>
```

Comment:
Warning: `rfind n/` works: No keywords so it finds 0 reviews.

Reply:
Also, there are review find commands, not stall find commands.

Reply:
Fixed.